src/controllers/trip.controller.js

```diff
@@ -66,6 +66,27 @@ class TripController {
     ResponseService.setSuccess(200, 'List of requested trips', trips);
     return ResponseService.send(res);
   }
+
+  /**
+   * @param {req} req
+   * @param {res} res
+   * @returns {comments} this function create comments on trip request
+  */
+  static async userCommentOnRequestsTrip(req, res) {
```

Comment:
can you please change to `addCommentOnTripRequest` or `createTripRequestComment`